index.bs

```diff
@@ -6366,9 +6366,11 @@ steps may throw an exception.
       <!-- Binary -->
       : If |input| is a [=buffer source type=]
       ::
+          1. If [$IsDetachedBuffer$](|input|) is true then return invalid.
```

Comment:
IsDetachedBuffer() takes an ArrayBuffer or a SharedArrayBuffer, but buffer source type includes the views. I think you'll need to dig the buffer out of the view in those cases?

Reply:
Thanks for the review!  I found the new steps from the WebIDL spec here:

https://webidl.spec.whatwg.org/#buffersource-detached

Reply:
I updated the latest revision to use the shorthand for these steps.